core/templates/services/UpgradedServices.ts

```diff
@@ -683,6 +683,7 @@ interface UpgradedServicesDict {
 export class UpgradedServices {
   getUpgradedServices(): UpgradedServicesDict {
     var upgradedServices = {};
+    /* eslint-disable-next-line oppia/no-multiline-disable */
```

Comment:
Ditto, please do not do this.

Reply:
Ah ok -- understood. Perhaps add this info as a comment. It might be OK for this file since @srijanreddy98 is going to get rid of it, but let's double-check with him.

@srijanreddy98: do you know when this file is planned to be removed, and (b) is there a way to write it by which we can avoid the lint errors here?

Reply:
Done.